### DISCUSSION TITLE
Why are Stop-AdfTriggers and Start-AdfTriggers asymmetric?

### DISCUSSION THREAD

**Discussion by camilo-s** [Azure-Player/azure.datafactory.tools#313] [Q&A]
Hi @NowinskiK, first of all thank you for the great library!!

Is there a reason why the two public functions `Stop-AdfTriggers` and `Start-AdfTriggers` are _asymmetric_ in that the former receives strings `FactoryName` and `ResourceGroupName` as parameters, whereas the latter receives an ADF object? 

As consequence of this, `Stop-AdfTriggers` can't receive a `PublishOptions` attribute, so the [logic](https://github.com/Azure-Player/azure.datafactory.tools/blob/db441e26237fe61e594b4b51b84ea362d06e3837/private/Stop-Triggers.ps1#LL24C16-L24C16) to selectively stop Triggers in `Stop-Triggers` doesn't come to effect.

**Comment by NowinskiK** [comment]
This is because:
* `Stop-AdfTriggers` is happening base on target ADF service, whereas
* `Start-AdfTriggers` uses code-ADF (just deployed) to identify which triggers should be started.
Therefore, there is no reason to use `PublishOptions` in order to select triggers to be disabled. 
`PublishOptions` is reserved for publishing operation only.
